### ISSUE TITLE
Home Page

### ISSUE THREAD

**Issue by lynn-gavett** [megandalster/PQG-Website#13]
Please move the Menu Bar below the Quilt Photos, and add a box around the Menu Bar that can be outlined with a color or filled with a color.

**Comment by megandalster** [comment]
I cannot easily switch these two around, but I can move the slider to the very top of the site.  What do you think about that variation?

**Comment by lynn-gavett** [comment]
Not going to work, sorry.  Let's put a box around the Menu, and move it up closer to PQG.  And move the Quilt Photos up, as well.  Aiming for less white space.

**Comment by megandalster** [comment]
I did the best I could for now... still working on this.  Take a look.

**Comment by megandalster** [comment]
The Home page now has a full description of the next upcoming Membership Meeting and Workshop, but it shows in two instances.  I did this to open a discussion about how to manage events on the web site going forward.

The first instance is the one that I added -- it has two descriptions of these two events as they appear on the upcoming events list on the right.  Clicking on either one gives you the full description of that event, including images.  (I have added new Editor Help instructions show you how to create a new Membership Meeting event and a new Workshop event.)  

The second instance is the one that you had added, with the September membership meeting and workshop merged into a single text.  

If we think of a PQG event as one that can be archived for future reference, even after its time has gone by, then it will be a lot less work to post events on the Home page using the first approach than the second.  For instance, thinking forward into October and November, what happens to these two September events on the web site.  In the first instance, they remain on the calendar forever, and can be recalled instantly with no extra work.  In the second instance, they may be may be saved off-line with some extra work, but typically will be lost since they were never put on the calendar in the first place.

So I would vote for the first approach.  Let me know what you think and if I have overlooked something.

PS Still working on the box around the menu...

**Comment by megandalster** [comment]
Added light blue color to background in menu bar so that it matches upcoming events on the right.  You can change this the same way you changed the background color on upcoming events.

**Comment by lynn-gavett** [comment]
I understand what you are saying.  The problem is the appearance of the Event on the Home Page.  I tried to add the copy we would want to see on the Event edit page, and the photos won't move where I want them, and the font and colors do not show up on the Home Page.  We need to talk about this, and the overall appearance of the Home Page.

**Comment by megandalster** [comment]
I see what you mean.  I need to fix the styling of the events so that they appear with photos, coloring, and fonts that you want on the home page.  Once that is done, all future events will reflect the new styling.  Any details you can give me on the styling (font, font color, type size, etc) will help me with that fix.  I haven't figured out how to do this yet but I will. 

Beyond that, what more should be done with the overall appearance of the home page?  Should we start over with a new design?

**Comment by lynn-gavett** [comment]
Heavens NO! Let's try the following:
Downsize the PQG logo and PQG font.
Bring the Menu up as tight as you can to the Logo frame, make the Menu frame run from edge to edge of the page, and outline in red.  I removed the background color.
Bring the Photos as tight as you can to the Menu frame.  (I am trying to make the photos the predominant feature on the page.
Move the Upcoming events to the right side of the page edge.  Outline frame in red. I removed the background color.
Outline the Copyright, etc. frame in red.  This frame should be tight to the bottom of the page.
Let's start with the Home page styling, then we can discuss the events links.

**Comment by lynn-gavett** [comment]
The red boxes do not line up, and appear differently depending on the device used to open the site, so how about deleting the boxes, and just use lines that run from side to side.  I am sending you a pdf.file that I mocked up in Publisher for reference.

**Comment by megandalster** [comment]
I didn't see the PDF, but I have been working a lot on the home page.  Hopefully it is more visually attractive than it was.
